### DISCUSSION TITLE
Web browser engine

### DISCUSSION THREAD

**Discussion by RVenkatesh** [responsively-org/responsively-app#592] [Q&A] [answered]
I am new to this application. I am trying to understand how much Responsively can be used for testing  the behaviour of our apps for various devices.

Questions in that aspect are as follows.

1. Does responsively use different browser engines like Webkit, Gecko, Blink and so on?
2. If only one engine is being used, which engine is it?

**Comment by manojVivek** [comment, marked as answer]
@RVenkatesh Responsively doesn't use different browser engines for devices. It uses Chromium as the browser, so `Blink` is the engine used.

**Reply by RVenkatesh** [reply]
@manojVivek Thanks for the reply.

**Reply by gennahumpal** [reply]
Msi labtop linked to note 20 ultra phone doing 2007 chevy pickup 5.3 diagnostics for no crank no start?
